Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <Authors>Sentry Team and Contributors</Authors>
-    <Version>1.5.3</Version>
+    <Version>1.5.4</Version>
```

Comment:
Running it as such to see if error rate changes. Will bump ++ if the change makes sense